src/schema/organizations.ts

```diff
@@ -711,6 +714,8 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
                 },
               ),
           ]);
+
+          notifyOrganizationUserJoined(ctx.log, member.id, organization.id);
```

Comment:
We can probably make the functions accept a single param of object. Sometimes the order gets confusing when both types are string and might get mixed up. At least with object, it is a bit harder to miss.

Reply:
Good point, will add